### ISSUE TITLE
Don't try to show HTP annotations in Canto

### ISSUE THREAD

**Issue by kimrutherford** [pombase/canto#2702]
(From pombase/curation#3465)

Add a message like this when there are existing HTP annotations for a paper:

```
This publication has NNNN existing annotations from high throughput experiments.
These are not displayed in Canto.
Please visit the publication page for PMID:1234567 at pombase.org to view them.
```

and make `publication page for PMID:1234567` link to the publication page.

(But improve the wording)

**Comment by kimrutherford** [comment]
I've changed the code so that HTP existing annotations aren't shown.

> Add a message like this when there are existing HTP annotations for a paper:

That's a bigger task so will take a bit more work.

**Comment by ValWood** [comment]
what is the outstanding task here?

**Comment by kimrutherford** [comment]
This bit isn't done:

> Add a message like this when there are existing HTP annotations for a paper:

**Comment by ValWood** [comment]
Is this ticked needed? https://github.com/pombase/canto/issues/1327

**Comment by kimrutherford** [comment]
> Is this ticked needed? #1327

Thanks.  I closed it.

**Comment by kimrutherford** [comment]
> is this just adding the message?

Yep.  But it needs a new internal API for Canto to query Chado to see if there are HTP annotations for the current paper.
